sunrisemvmtnyc/legislation-tracker
# feat: Advanced page / legislative sheet

Eventual goal is to look like the [google sheet](https://docs.google.com/spreadsheets/d/1oTbQNLVf8iMC0ShOdkilr3Sh9d4-Rf8Gjt0ZyU2Gq18/edit#gid=378151632). I'm already at 300 lines of code, so I'll do more in a separate PR. `fixme`s announce what remains to work on.

### To test:
Visit the "advanced" page. Verify that you see the members, bills, and whether/not they are sponsoring.
